flask-redis-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,31 @@
+---
+specversion: 0.0.2
+id: flask_redis_nulecule
+
+metadata:
+    name: Flask Redis Nulecule App
+    appversion: 0.0.1
+    description: Atomic app for deploying Flask + Redis page visit counter
+graph:
+    - name: redis
+      params: 
+        - name: redisimage
+          description: Redis image
+          default: dharmit/redis
+      artifacts:
+        docker:
+          - file://artifacts/docker/redis
+        kubernetes:
+          - file://artifacts/kubernetes/00-service-redis.yaml
+          - file://artifacts/kubernetes/01-pod-redis.yaml
```

Comment:
The artifacts filenames do not necessarily start with numbers stating the order in which objects to be created. They can be simple as "redis-service.yaml" , "redis-rc.yaml"